input/fsh/CodeSystem-Omim.fsh

```diff
@@ -0,0 +1,508 @@
+CodeSystem: Omim
+Id: omim
+Title: "omim"
```

Comment:
Let's spell this out: `"Online Mendelian Inheritance in Man"`.

Reply:
done